spec/firebase_app_distribution_api_client_spec.rb

```diff
@@ -545,6 +545,21 @@
       expect(result).to eq(1)
     end
 
+    it 'returns the 0 if response is empty' do
```

Comment:
super nit: "returns the 0" -> "returns 0"

Reply:
Good catch! Done.